src/doc/index.md

```diff
@@ -45,8 +62,9 @@ accomplishing various tasks.
 
 <div>
   <form action="std/index.html" method="get">
-    <input type="search" name="search"/>
-    <button>Search</button>
+    <input id="search-input" type="search" name="search"
+           placeholder="Search into the standard library!"/>
```

Comment:
I'd prefer something like "Search through the standard library" (without the exclamation mark), though i'm welcome to a second opinion from @rust-lang/docs.

Reply:
Yes sir!